.storybook/helpers/breakpoint-viewports.js

```diff
@@ -0,0 +1,14 @@
+import * as breakpoints from '../../src/design-tokens/breakpoint.yml';
+
+/**
+ * List of viewports to use with addon-viewport, based on Theo breakpoints tokens
+ */
+export const breakpointViewports = Object.keys(breakpoints).map((name) => {
+  return {
+    name,
+    styles: {
+      width: breakpoints[name],
+      height: '100%',
+    },
```

Comment:
https://www.npmjs.com/package/@storybook/addon-viewport#viewport-model

The Viewport Model indicates `type` ('desktop' | 'mobile' | 'tablet' | 'other') is not optional, but without strict typing, nothing here is broken. I wonder whether we need to indicate what `type` each of these is supposed to represent or whether we should just ignore that property.